framework/origin_trials_client.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from dataclasses import asdict
+from typing import Any
+import requests
+
+from framework import secrets
+from internals.data_types import OriginTrialInfo
+import settings
+
+
+
+# Data type for information if an error occurred. Error code + message.
+ERROR_INFO_TYPE = tuple[int, str]|None
```

Comment:
```suggestion
```

Reply:
Sorry, I remember removing the code for the error returns and the type hinting, but it doesn't show up here. It might have been part of an abandoned change. Either way, I should have caught it before sending it over for re-review!